proto/stream.proto

```diff
@@ -1,17 +1,23 @@
-syntax = "proto2";
+syntax = "proto3";
 
 package pb;
 
-import "metadata.proto";
-import "source.proto";
-
+import "fee.proto";
+import "file.proto";
 
 message Stream {
-    enum Version {
-        UNKNOWN_VERSION = 0;
-        _0_0_1 = 1;
-    }
-    required Version version = 1;
-    required Metadata metadata = 2;
-    required Source source = 3;
+    bytes hash = 1;
+    string language = 2;
+    string title = 3;
+    string author = 4;
+    string description = 5;
+    string media_type = 6;
+    string license = 7;
+    File file = 8;
+
+    Fee fee = 16;
+    string license_url = 17;
+    string thumbnail_url = 18;
+    uint32 duration = 19;
+    repeated string tags = 20;
```

Comment:
Can tags have a "type" property the way it's defined currently? i.e. Category/topic/genre/other/etc.

Reply:
i'm not sure what you mean. can you give me some examples?

Reply:
Per the discussion this afternoon, this would pose UX challenges when users tagged content. 

The idea was that a top level tag might be bound to some type, i.e. category, with predefined values. Then users could easily reference the category tag when searching/selecting based on tags, as opposed to more generic tags.